app/build.gradle

```diff
@@ -72,12 +72,6 @@ android {
         coreLibraryDesugaringEnabled = true
     }
 
-    lintOptions {
```

Comment:
Please leave this gradle file alone unless making a PR explicitly to edit it, or if it's needed for the code changes (in this case it's not needed)